wwwroot/index.html

```diff
@@ -6,14 +6,16 @@
     <link rel="icon" type="image/png" href="favicon.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
     <title>sourcegen.dev - Source Generator Playground - @davidwengier</title>
+    <meta name="description" content="Explore Roslyn source generators online" />
+    <meta name="keywords" content="roslyn .net source generators playground" />
     <base href="/" />
     <link href="css/bootstrap/bootstrap.min.css" rel="stylesheet" />
     <link href="css/app.css" rel="stylesheet" />
     <link href="_content/BlazorMonaco/lib/monaco-editor/min/vs/editor/editor.main.css" rel="stylesheet" />
 </head>
 
 <body>
-    <app>Loading...</app>
+    <app>Source Generator Playground. Loading...</app>
```

Comment:
Not sure how you feel about this one, just trying to add more context for search engines.